Only store messages if the guild has a message log set (#847)
ClemBotProject/ClemBot: ClemBot.Docs/docs/DesignatedChannels.md

| user_join_log  | Channel for ClemBot to log all users that join the server                             |
| user_leave_log | Channel for ClemBot to log all users that leave the server                            |

:::note

> Enabling the message_log channel causes ClemBot to store the content of messages for a period of 30 days. After which

Reviewer: ```suggestion
Enabling the `message_log` channel causes ClemBot to store the content of messages for a period of 30 days, after which, they are deleted.
```

> they are deleted. This is to enable ClemBot to notify server staff of message edits or deletions that might not be stored

Reviewer: ```suggestion
This enables ClemBot to notify server staff of message edits or deletions that might not be stored
```

Author: this is deleing the end of the first sentence

> in the bots short lived cache. Any edits or deletions that occur on messages that are older then 30 days will be sent in 

Reviewer: ```suggestion
in the bots short-lived cache. Any edits or deletions that occur on messages that are older then 30 days will be sent in 
```
